CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Change Log
 
+## v6.1.0 (Unreleased)
+
+### Added
+
+-   Added `testID` and `accessibilityLabel` to `<InfoListItem>` for easy access in UI and E2E tests.
```

Comment:
feels a bit strange to only add those to one component and publish a minor version out of it. Are we going to publish those as beta and add those to all our components?